dex/version/version.go

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) 2013-2014 The btcsuite developers
+// Copyright (c) 2015-2022 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
```

Comment:
Not sure what is best here. The LICENSE file in our root directory is a different license. Should either include the LICENSE file this is referencing or include a link or change this to ours. Not sure at all... And what if we change this file will we just up the date?

Reply:
Part of me wants to just change this to the blueoak license but I suspect it's more complicated than that even though we are the "Decred developers".  So we could do one of the following:
- a link to the license in the dcrd repo (https://github.com/decred/dcrd/blob/master/LICENSE)
- the actual 15 or so lines of the license from that file in this comment, just without the Lightning Network line since that does not pertain to this file
- put a LICENSE file in this dex/version folder

I think a URL would be just fine personally.